client/state/selectors/test/get-current-plan-term.js

```diff
@@ -0,0 +1,61 @@
+/** @format */
+
+/**
+ * Internal dependencies
+ */
+
+import * as plansLib from 'lib/plans';
```

Comment:
We seem to be using just `getPlan` from `plansLib`, so maybe import just it?

Reply:
If we keep this import as-is, it makes mocking with `jest.spyOn`, which has the benefit of allowing [`jest.restoreAllMocks()`](https://facebook.github.io/jest/docs/en/jest-object.html#jestrestoreallmocks) in `beforeEach`, which I would encourage here. See https://github.com/Automattic/wp-calypso/pull/23958#discussion_r180321578

I'd actually encourage that approach with `getSitePlan` as well.